### PR TITLE
SW-1818 Allow user to apply viability percent from cut test on the accession.

### PR DESCRIPTION
**Pull request by karthikbtf**
- moved viability percent calculation to a util
- removed cut tests seedsTested validation (since this is implied and not user supplied)

### CODE REVIEW

File: src/components/accession2/viabilityTesting/NewViabilityTestModal.tsx [hunk 1]
```diff
@@ -221,7 +221,7 @@ export default function NewViabilityTestModal(props: NewViabilityTestModalProps)
       if (response.requestSucceeded) {
         reload();
         onCloseHandler();
-        if (testCompleted && !readOnly) {
+        if ((testCompleted || record.testType === 'Cut') && !readOnly) {
```
Comment (constanzauanini) [inline src/components/accession2/viabilityTesting/NewViabilityTestModal.tsx:224]: In cut test readOnly will always be false (we can't mark it as complete), so if we are editing the cut test and not changing any value we will see the viability message again. Not sure if that's correct

Comment (karthikbtf) [reply src/components/accession2/viabilityTesting/NewViabilityTestModal.tsx:224]: good catch, i will update this to only check when the value is edited

Comment (karthikbtf) [reply src/components/accession2/viabilityTesting/NewViabilityTestModal.tsx]: updated and tested

**Review by constanzauanini — APPROVED** [review]
🎉

**Pull request merged**